### ISSUE TITLE
Issue with number of classes and partial predictions in PSPNet

### ISSUE THREAD

**Issue by sainatarajan** [divamgupta/image-segmentation-keras#123]
Hi! Thanks for this easy to use repo. I am running a PSPNet50 model that has 4 labels including the background. After a good amount of training (100 epochs), only 2 labels (background and 1 more label) get predicted. Can you tell me what is the issue here? Am I defining the model correctly? Do I need to set `n_classes= 4` or `n_classes= 3`? Or the masks have to colored using a predefined color list?

```
model = keras_segmentation.models.pspnet.pspnet_50(n_classes=4)
model.train( 
    train_images =  ROOT_DIR+"train_images/",
    train_annotations = ROOT_DIR+"train_masks/",
    checkpoints_path = ROOT_DIR+"checkpoints/" , epochs=100
)
```

**Comment by divamgupta** [comment]
Try fine-tuning from the PSPNet pre-trained model.
